src/implementations/twig/components/page-header/page-header.html.twig

```diff
@@ -0,0 +1,117 @@
+{% spaceless %}
+{#
+  Parameters:
+    - "variant" (string) (default: '') core, standardised or harmonised
```

Comment:
instead of having variant for the families, we should have two possible displays ("positive" and "negative")
then one of the other will be used to create a family specific page header